### PR TITLE
Asset - deployment component

### PR DESCRIPTION
**Pull request by HrishikeshGeedMS**
Deployment component assets PR
This component will be used for deploying model using online endpoint

### CODE REVIEW

File: common/components/deploy_model/spec.yaml [hunk 1]
```diff
@@ -0,0 +1,187 @@
...
+version: 0.0.1
+type: command
+
+is_deterministic: True
+
+display_name: Deploy model
+description:
+  Deploy a model to a workspace. The component works on compute with [MSI](https://learn.microsoft.com/en-us/azure/machine-learning/how-to-create-manage-compute-instance?tabs=python) attached.
+
+environment: azureml:python-sdk-v2:1
```
Comment (vizhur) [inline common/components/deploy_model/spec.yaml:12]: Image for this environment was never released to MCR

**Review by vizhur — CHANGES_REQUESTED** [review]

File: common/components/deploy_model/spec.yaml (hunk 1 above)
Comment (skasturi) [reply common/components/deploy_model/spec.yaml:12]: Has the release completed, @novaturient95 ?

**Comment by skasturi** [comment]
> 

What is the structure of this JSON file? 

---
Refers to: common/components/deploy_model/spec.yaml:188 in 8c43f5a. [](commit_id = 8c43f5a607584e6f14b9d72afb92bfead35cf618, deletion_comment = False)

File: common/components/deploy_model/spec.yaml (hunk 1 above)
Comment (skasturi) [reply common/components/deploy_model/spec.yaml:12]: Release is done. Approving.

**Review by skasturi — APPROVED** [review]

File: common/components/deploy_model/spec.yaml (hunk 1 above)
Comment (vizhur) [inline common/components/deploy_model/spec.yaml:12]: 2

**Review by vizhur — APPROVED** [review]

**Pull request merged**